posthog-react-native/src/surveys/components/Surveys.tsx

```diff
@@ -47,8 +47,8 @@ export const dismissedSurveyEvent = (survey: Survey, posthog: PostHog): void =>
   posthog.capture('survey dismissed', {
     $survey_name: survey.name,
     $survey_id: survey.id,
-    $survey_iteration: survey.current_iteration,
-    $survey_iteration_start_date: survey.current_iteration_start_date,
+    $survey_iteration: survey.current_iteration ?? null,
```

Comment:
Oh I really want a `?` operator to do this automatically

Reply:
solved with a `maybeAdd` for now